api/libproject/devstart.rb

```diff
@@ -1348,3 +1348,25 @@ def setup_cloud_project(cmd_name, *args)
   :description => "Initializes resources within a cloud project that has already been created",
   :fn => lambda { |*args| setup_cloud_project("setup-cloud-project", *args) }
 })
+
+def start_api_and_incremental_build(cmd_name, args)
+  ensure_docker cmd_name, args
+  common = Common.new
+  begin
+    # appengineStart must be run with the Gradle daemon or it will stop outputting logs as soon as
+    # the application has finished starting.
+    common.run_inline %W{gradle --daemon appengineStart}
+    # incrementalHotSwap must be run without the Gradle daemon or stdout and stderr will not appear
+    # in the output.
+    common.run_inline %W{gradle --no-daemon --continuous incrementalHotSwap}
+  ensure
+    common.run_inline %W{gradle --stop}
+  end
+end
+
+# TODO(dmohs): This is really isn't meant to be run directly, so it'd be better to hide it from the
```

Comment:
Should we instead just inline it in docker-compose.yaml then?

Reply:
OK. In the meantime, do we want to add a description saying "don't run this directly"?

Reply:
Done.